.vitepress/config.js

```diff
@@ -0,0 +1,381 @@
+import simpleGit from 'simple-git';
+import fs from 'fs';
+import grayMatter from 'gray-matter';
+import { createLogger } from 'vite-logger';
+import { Feed } from 'feed';
+import { withPwa } from '@vite-pwa/vitepress';
+import { defineConfig } from 'vitepress';
+import yaml from 'js-yaml';
+
+import feedConfig from './feeds.js';
+
+const git = simpleGit();
+
+const logger = createLogger('info', true);
+
+const statuses = [ 'Living', 'Last Call', 'Final', 'Review', 'Draft', 'Withdrawn', 'Stagnant' ]
+const eips = Promise.all(fs.readdirSync('./EIPS/').map(async file => {
+    let eipContent = fs.readFileSync(`./EIPS/${file}`, 'utf8');
+    let eipData = grayMatter(eipContent);
+    let lastStatusChange = new Date((await git.raw(['blame', `EIPS/${file}`])).split('\n').filter(line => line.match(/status:/gi))?.pop()?.match(/(?<=\s)\d+-\d+-\d+/g)?.pop());
```

Comment:
What is --porcelain?

Reply:
git has two types of commands: plumbing and porcelain. Plumbing commands are low-level and are used to implement the higher level porcelain commands, while porcelain commands are made to interface with a human.

The `--porcelain` flag tells porcelain commands to output in a more stable and script friendly format.

Reply:
> I know we disallow colons in eipw, but if we change that, we should still expect this tool to work. I'd recommend using /^status:/gi as the regex in case someone puts something dumb in their title.

How does that break this line?